unix/snowagent.config

```diff
@@ -83,22 +92,23 @@
             <Path>*.pl</Path>
             <Path>*.png</Path>
             <Path>*.po</Path>
-            <Path>*.properties</Path>
             <Path>*.rdf</Path>
             <Path>*.sdl</Path>
             <Path>*.so</Path>
             <Path>*.sql</Path>
             <Path>*.ttf</Path>
             <Path>*.txe</Path>
-            <Path>*.txt</Path>
             <Path>*.utf8</Path>
-            <Path>*.xml</Path>
             <Path>*.xpt</Path>
             <Path>*.zip</Path>
+			<!-- *.xml *.txt and *.properties files would normally be excluded but are required for SIOMS -->
+			<!--<Path>*.xml</Path>
+			<Path>*.txt</Path>
+			<Path>*.properties</Path>-->
```

Comment:
Left it commented in case someone wants to enable the exclusion of those files.
I see it as those placeholder system settings that we have commented. Just easier access.